clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -1830,24 +1833,52 @@ public DescribeClusterResult describeCluster(DescribeClusterOptions options) {
         runnable.call(new Call("listNodes", calcDeadlineMs(now, options.timeoutMs()),
             new LeastLoadedNodeProvider()) {
 
+            private boolean useMetadataRequest = false;
```

Comment:
This seems fine at the moment since this isn't particularly performance critical. For the longer term if we have many of these downgrades, we could consider using ApiVersions in the underlying NetworkClient to make the decision before sending the request.

Reply:
Yeah, that's a good idea.